docs/api.md

```diff
@@ -0,0 +1,59 @@
+# Armada API
+
+Armada exposes gRPC and REST api.
```

Comment:
`api` should be `API` (or possibly `APIs`, if we consider the gRPC and REST endpoints to be separate). Possibly "Armada exposes an API via gRPC or REST"?